### PR TITLE
[DEV-4219] Update README

### PR DESCRIPTION
**Pull request by djui**
And add license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,122 @@
...
-discussed here as we don't have access to physical hardware or want to invest
-into a NAT proxy e.g. on AWS.
+discussed here as it more convenient nowadays to not require access to physical
+hardware or want to invest into a NAT proxy e.g. on AWS.
 
 
-### Forwarding Proxy
+#### Forwarding Proxy
 
 A forwarding proxy can come in two flavours:
```
Comment (jeespers) [inline README.md]: The row below needs a reformulation.

File: README.md [hunk 2]
```diff
@@ -1,91 +1,182 @@
...
+  -user string
+    	Server authentication username
+  -verbose
+    	Set log level to DEBUG
+```
+
+To start the proxy as HTTP server, just run:
+
+```
+$ forwardingproxy -cert cert.pem -key key.pem
```
Comment (alesr) [inline README.md]: I guess you forgot to remove the certificates from this command. Otherwise, you'll start the HTTPS server.

Comment (djui) [reply README.md]: c'n'p. Will fix.

Comment (djui) [reply README.md]: https://github.com/betalo-sweden/forwardingproxy/commit/6fe37f5b90848d9359d21d62b2427acebd9fda7f

**Pull request merged**